server/service/odp/deleteDraft.ts

```diff
@@ -1,3 +1,20 @@
-import { OdpRepository } from '@server/repository/odp'
+import { User } from '@core/auth'
+import { BaseProtocol, DB } from '@server/db'
+import { getAndCheckOdpCountryId } from '@server/repository/odp/getAndCheckOdpCountryId'
+import { OdpClassRepository, OdpRepository } from '@server/repository'
 
-export const { deleteDraft } = OdpRepository
+export const deleteDraft = async (options: { odpId: number; user: User }, client: BaseProtocol = DB) => {
```

Comment:
methods should return Promise<whatever>
It can be done later if you wish so

Reply:
Separate PR